backend/engine/utils/plugin.py

```diff
@@ -366,7 +375,7 @@ def run_plugin(plugin, scan, scan_images, depth=None, include_dev=False, feature
     )
 
 
-def process_event_info(scan, results, plugin_type, plugin_name):
+def process_event_info(scan: Scan, results, plugin_type: str, plugin_name: str):
```

Comment:
I see the param `results` does not have an expected type, is it because any data type can be expected here that can be expected here?

Reply:
Yeah, I left some params as untyped if I couldn't determine with certainty the possible types.